src/components/molecules/ExecutionDetails/TestSuiteExecutionDetails/TestSuiteExecutionDetailsTabs.tsx

```diff
@@ -1,32 +1,32 @@
 import {useContext} from 'react';
 
+import {Tabs} from 'antd';
+
+import {Tab} from 'rc-tabs/lib/interface';
+
 import {TestSuiteExecution} from '@models/testSuiteExecution';
 
 import {ExecutionStepsList} from '@molecules';
 
 import {ExecutionDetailsContext} from '@contexts';
 
-import {
-  StyledAntTabPane,
-  StyledAntTabs,
-  StyledTestSuiteExecutionDetailsTabsContainer,
-} from '../ExecutionDetails.styled';
-
 const TestSuiteExecutionDetailsTabs: React.FC = () => {
   const {data} = useContext(ExecutionDetailsContext);
 
   const testSuiteData = data as TestSuiteExecution;
   const {stepResults} = testSuiteData;
 
-  return (
-    <StyledTestSuiteExecutionDetailsTabsContainer>
-      <StyledAntTabs>
-        <StyledAntTabPane tab="All Steps" key="AllStepsPane">
-          {stepResults ? <ExecutionStepsList executionSteps={stepResults} /> : null}
-        </StyledAntTabPane>
-      </StyledAntTabs>
-    </StyledTestSuiteExecutionDetailsTabsContainer>
-  );
+  const items = [
+    stepResults && stepResults.length
```

Comment:
Shouldn't this tab be empty but visible?

Reply:
Meaningful